dev/bots/analyze.dart

```diff
@@ -129,7 +129,7 @@ Future<void> run(List<String> arguments) async {
 // TESTS
 
 final RegExp _findDeprecationPattern = RegExp(r'@[Dd]eprecated');
-final RegExp _deprecationPattern1 = RegExp(r'^( *)@Deprecated\($'); // ignore: flutter_deprecation_syntax (see analyze.dart)
+final RegExp _deprecationPattern1 = RegExp(r'^( *)@Deprecated\($');
```

Comment:
this one isn't intended for the analyzer, it's intended for the analyze.dart script. we need some alternative syntax to allow this file to know which deprecations to ignore when it's doing its scan.